src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -512,4 +512,26 @@ public static boolean isName(String str) {
 
         return isName;
     }
+
+    /**
+     * Returns true if the value arg is either null, or empty
+     * or contains only whitespace characters.
+     * More efficient that calling (string).trim().length() == 0
+     * or (string).trim().isEmpty()
+     * because it does not create temporary string whereas trim() does
+     * @param value A string to check
+     * @return true if the value does not contain any character except whitespace.
```

Comment:
@romani @Vladlis @rnveach @MEZk 
This class already has method isBlank(String str)
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java#L434